src/wasm/wasm.cpp

```diff
@@ -1631,7 +1631,7 @@ Importable* Module::getImport(ModuleItemKind kind, Name name) {
 
 Importable* Module::getImportOrNull(ModuleItemKind kind, Name name) {
   auto doReturn = [](Importable* importable) {
-    return importable->imported() ? importable : nullptr;
+    return importable ? importable->imported() ? importable : nullptr : nullptr;
```

Comment:
Odd this wasn't an issue before...

Reply:
yeah; actually this part ended up not being needed for this PR (since the Name of the import object in Binaryen isn't the imported name of the table?), but I figured it would be best to leave the fix anyway.

Reply:
The double ternary is a little confusing. Can it be refactored as a single ternary?